package.json

```diff
@@ -87,7 +90,6 @@
     "recompose": "^0.21.2",
     "redux": "^3.5.2",
     "reselect": "^2.5.4",
-    "uswds": "^1.6.3",
-    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema.git#86263a396bfe2b13153797bac78c424a913b450d"
+    "uswds": "^1.6.3"
```

Comment:
Probably could move this to peer dependencies as well.

Reply:
I can just make that change quickly now!